repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/guidance-content.ftl

```diff
@@ -4,8 +4,28 @@
 <#macro guidance guidanceDocument showTitle=true>
 <#-- @ftlvariable name="guidanceDocument" type="uk.nhs.hee.web.beans.Guidance" -->
     <#if guidanceDocument??>
+        <@hst.link var="heroImage" hippobean=guidanceDocument.heroImage />
+        <#assign showHero=false>
+        <#if heroImage??>
+            <#assign showHero=true>
+            <section class="nhsuk-hero nhsuk-hero--image nhsuk-hero--image-description" style="background-image: url('${heroImage}');">
```

Comment:
We might need clarity from the design team on where to display hero image when rendering under mini-hub as it is currently being displayed below `Contents` section.

Reply:
will inform authoring that hero images aren't supported in minihub area.